mlrun/api/api/endpoints/model_endpoints.py

```diff
@@ -123,7 +180,9 @@ def list_endpoints(
 ) -> ModelEndpointList:
```

Comment:
Neither do I since it's a `BaseModel` object that contains only a single field `endpoints` which is literally a list of model endpoints... 
Perhaps the `ModelEndpointList` was created for other reasons (that were not implemented) such as grouping several model endpoints together.